analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -80,11 +80,10 @@ class Bundler(
         ) = Bundler(managerName, analysisRoot, analyzerConfig, repoConfig)
     }
 
-    override fun command(workingDir: File?) = if (Os.isWindows) "bundle.bat" else "bundle"
+    override fun command(workingDir: File?) = "ruby"
```

Comment:
Commit message: I'm a bit surprised that the code around using `bundler_dependencies.rb` is not removed. Is it still used? If so, could you please explain in the commit message how `bundler_dependencies.rb` and `bundler_dependencies_metadata.rb` play together now?

Reply:
Yes, I will add the reason to the commit message. Short explanation: This script is still useful to get top level dependencies
and scopes. Would probably also be possible with the meta data obtained by the new script, but I decided to keep the short script.